src/lazyload.setSources.js

```diff
@@ -20,6 +20,10 @@ export const setSources = function (element, settings) {
     const {data_src: dataSrc, data_srcset: dataSrcSet} = settings;
     const tagName = element.tagName;
     const elementSrc = getData(element, dataSrc);
+    if(!src) {
```

Comment:
I can't understand how can `(!src)` be ever true, here, since that variable is not declared anywhere.